src/injector.js

```diff
@@ -0,0 +1,103 @@
+const puma = require('pumascript');
+const http = require('http');
+const fs = require('fs');
+
+
+let errorsLogs = [];
+
+
+function readDependencyJsonFile(dir) {
+    try {
+        return JSON.parse(fs.readFileSync(dir, 'utf8'));
+    } catch (e) {
+        console.log('The file entered does not exist');
+    }
+    return undefined;
+}
+
+
+function readUrl(options, callback) {
+    var req = http.get(options, function (res) {
+        var bodyChunks = [];
+        res.on('data', function (chunk) {
+            bodyChunks.push(chunk);
+        }).on('end', function () {
+            var body = Buffer.concat(bodyChunks);
+            callback(body);
+        })
+    });
+    req.on('error', function (e) {
+        console.log('ERROR: ' + e.message);
+    });
+}
+
+
+
+function createFile(dir, fileName, data) {
+    fs.writeFileSync(dir + '/' + fileName, data);   
+}
+
+//Error object
+function Error(url) {
+    this.url = '' || url;
+    this.listErrors = [];
+}
+Error.prototype.addError = function (line, colum, componentGeneratesError) {
+    this.listErrors.push({
+        line, colum, componentGeneratesError
+    })
+}
+
+
+function testPuma(dataUrl, url) {
+    console.info('********** Entering ', url, '****************************');
+    var result = puma.evalPuma(dataUrl, 'test');
+    let error = new Error();
+
+    error.listErrors = [];
+    if (result.success !== undefined) {
```

Comment:
if(result.sucess) should be enough

Reply:
if (result.sucess) is not enough because there are times where the result.sucess returns true or false, but there are cases where it is undefined.
Example:
cdnjs.cloudflare.com/ajax/libs/twitter-bootstrap/4.0.0-beta/js/bootstrap.js
cdnjs.cloudflare.com/ajax/libs/bootstrap-datepicker/1.7.1/js/bootstrap-datepicker.js
cdnjs.cloudflare.com/ajax/libs/angular-ui-bootstrap/2.5.0/ui-bootstrap.js
code.jquery.com/jquery-2.2.4.js
cdnjs.cloudflare.com/ajax/libs/jqueryui/1.12.1/jquery-ui.js
cdnjs.cloudflare.com/ajax/libs/angular.js/1.6.5/angular.js

in all these cdn result.sucess is equal to undefined